src/utils/evalUtils.js

```diff
@@ -17,9 +17,11 @@ const logger = require('winston');
 const errors = require('../errors');
 const evalValidation = require('./evalValidation');
 const sampleSchema = require('./schema').sample;
+const u = require('./commonUtils');
 const ERROR_MESSAGE = {
   TRANSFORM: {
     NOT_ARRAY: 'The transform function must return an array.',
+    NOT_OBJECT: 'The transform function must return an object.',
```

Comment:
we should just pull out all the "array" stuff for transform... right?

Reply:
I don't understand what you mean, but this was added in an earlier version when I thought I would have to support a transform that returned individual samples. It's not being used anymore, I'll remove it.